src/zino/events.py

```diff
@@ -192,9 +192,13 @@ def _delete(self, event: Event):
 
     def delete_expired_events(self):
         """Deletes all events that have been closed for a certain time"""
-        event_list = list(self.events.values())
-        for event in event_list:
-            if event.state == EventState.CLOSED and now() > (event.updated + EVENT_EXPIRY):
+        closed_events = list(event for event in self.events.values() if event.state == EventState.CLOSED)
+        for event in closed_events:
+            if not event.closed:
+                # Fixup close time for events that were closed but never had a close time set, to ensure it will
+                # expire eventually
+                event.closed = now()
+            if now() > (event.closed + EVENT_EXPIRY):
```

Comment:
```suggestion
            elif now() > (event.closed + EVENT_EXPIRY):
```

Since we know that the others are just set to `now` and therefore don't need to check them
But disregard this if you think the other way is clearer

Reply:
Good point, did not consider this, then your approach is better!